source/_integrations/fritzbox.markdown

```diff
@@ -32,29 +32,36 @@ There is currently support for the following device types within Home Assistant:
 
 ## Setup
 
+Go to the integrations page in your configuration and click on **new integration** -> **AVM FRITZ!Box**. If you have enabled SSDP discovery, it’s likely that you just have to confirm the detected device with username and password.
+
+### YAML configuration
+
+YAML configuration is still around for people that prefer YAML, but it's deprecated and you should not use it anymore.
+
 ```yaml
 # Example configuration.yaml entry
 fritzbox:
   devices:
     - host: fritz.box
-      username: YOUR_USERNAME
+      username: admin
```

Comment:
```suggestion
```

We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.